.github/workflows/codeql-build.yml

```diff
@@ -39,15 +28,11 @@ jobs:
 
       - name: Set up for build
         run: |
-          cp ./cfe/cmake/Makefile.sample Makefile
-          cp -r ./cfe/cmake/sample_defs sample_defs
-          make prep
-
+          cp Makefile.sample Makefile
+          make ENABLE_UNIT_TESTS=true PERMISSIVE_MODE=true prep
```

Comment:
Recommend either using all env settings or all specified on the make line, looks like at minimum ENABLE_UNIT_TESTS is set in both locations.

Reply:
Set PERMISSIVE_MODE in env settings and remove ENABLE_UNIT_TESTS in make command.